testeng/jobs/cleanupPythonCode.groovy

```diff
@@ -10,6 +10,11 @@ job('cleanup-python-code') {
         stringParam('repoNames', null, 'Comma or space separated list of names of (public) target repositories in $org org')
         choiceParam('pythonVersion', ['3.8', '3.5', '2.7'], 'Version of python to use')
         stringParam('packages', '', 'Comma or space separated list of packages to install (optional)')
+        stringParam('commitMessage', null, 'Commit message (will also be used as PR title).
+            See https://github.com/edx/open-edx-proposals/blob/master/oeps/best-practices/oep-0051-bp-conventional-commits.rst
+            for more information on best practices for commit messages')
+        textParam('prBody', '', 'Additional information for the PR body. You can add dynamic information to the
+        PR body by writing to .git/cleanup-python-code-description in the bash script (optional)')
```

Comment:
```suggestion
        stringParam('title', null, 'Commit message first line (will also be used as PR title).
            See https://github.com/edx/open-edx-proposals/blob/master/oeps/best-practices/oep-0051-bp-conventional-commits.rst
            for more information on best practices for commit messages')
        textParam('body', '', 'Additional information for the commit message and PR body.
        The script and packages will be included automatically.
        You can add dynamic information to the
        commit/PR body by writing to .git/cleanup-python-code-description in the bash script (optional).')
```
^ What do you think about breaking it up into title/body like this?

Reply:
I'm still not convinced including the PR body in the commit message is a good idea. In the upcoming use case, between the output of the requires.txt files and the actual script, that will be a huge commit message.

Reply:
Hmm, no, you're right -- the dynamic output could be huge. :thinking:

For the script, though, can we do something like this?

```
curl -sS "https://raw.githubusercontent.com/edx/edx-cookiecutters/2299c8e9c570155a4ea28c506dab4f7d22b3a87a/scripts/update_setup_py.sh" > update_setup_py.sh
bash update_setup_py.sh
```

Reply:
oh cute. i'll try that.

Reply:
It worked! The PR description is still long enough that I'd be worried about putting it in a commit message though

Reply:
Yeah, agreed, it could be really large. But it could also be too large for a PR description, too.

One possibility is to use a details/summary block for the PR body, like so:

<details>
<summary>Possibly giant chunk of output</summary>

```
output
output
output
```
</details>

I guess that could be either done by this PR-creator job or by whatever script it runs.